MVCDemos/Views/DropDownButton/Overview.cshtml

```diff
@@ -19,26 +19,35 @@
 
     <div class="dx-field">
         <div class="dx-field-label">
-            Custom main button action
+            Custom template and actions
         </div>
         <div class="dx-field-value">
             @(Html.DevExtreme().DropDownButton()
+                .ID("custom-template")
                 .SplitButton(true)
-                .Text("Sandra Johnson")
-                .Icon("../../Content/Images/gym/coach-woman.png")
                 .DisplayExpr("Text")
                 .KeyExpr("Value")
                 .UseSelectMode(false)
                 .DataSource(Model.ProfileSettings)
                 .OnButtonClick("profileButton_click")
                 .OnItemClick("profileItem_click")
+                .Template(@<text>
+                    <div class="button-img-container">
+                        <div class="button-img-indicator"></div>
+                        <img class="button-img" src="../../Content/Images/employees/51.png" alt="employee" />
```

Comment:
i can see that other images added as a resource to the project's file? like  <Content Include="Content\Images\employees\50.png" /> , but please check projects builds after adding it